.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -46,26 +60,24 @@ body:
       description: 'Please provide the following information about your environment; feel free to remove any items which are not relevant.'
       value: |
           - Operating system:
-          - If running via `appium` CLI...
-            - Appium CLI version (output of `appium --version`):
-            - Node.js version (output of `node --version`):
-            - `npm` version (output of `npm --version`): 
-            - Last `appium` version which did _not_ exhibit the problem: 
-          - If running Appium Desktop, its version:
-          - Appium driver(s) and their version(s):
-          - Appium plugin(s) and their version(s):
+          - Appium server version (output of `appium --version`):
+          - Appium driver(s) and their version(s) (if applicable):
+          - Appium plugin(s) and their version(s) (if applicable):
+          - Node.js version (output of `node --version`):
+          - `npm` version (output of `npm --version`):
+          - Last component(s) version which did _not_ exhibit the problem:
           - Platform and version under test:
-          - Real device or emulator/simulator:
+          - Real device or emulator/simulator (if applicable):
```

Comment:
I think we can remove `(if applicable)` from all of these since it's redundant with the description

Reply:
removed